content/hi/microservices.md

```diff
@@ -0,0 +1,20 @@
+| Title                                    | Status    | Category |
+| ---------------------------------------- | --------- | -------- |
+| [Microservices](https://github.com/cncf/glossary/blob/main/content/en/microservices.md) | Completed | Concept  |
+
+# यह क्या है
+
+```
+माइक्रोसर्विसेज अनुप्रयोग विकास के लिए एक आधुनिक दृष्टिकोण है जो क्लाउड नेटिव तकनीकों का लाभ उठाता है। जबकि आधुनिक एप्लिकेशन, जैसे नेटफ्लिक्स, एक एकल ऐप प्रतीत होते हैं, वे वास्तव में छोटी सेवाओं का एक संग्रह हैं - सभी एक साथ मिलकर काम कर रहे हैं। उदाहरण के लिए, एक एकल पृष्ठ जो आपको वीडियो तक पहुंचने, खोजने और पूर्वावलोकन करने की अनुमति देता है, संभवतः छोटी सेवाओं द्वारा संचालित होता है जो प्रत्येक इसके एक पहलू को संभालते हैं (जैसे आपके ब्राउज़र में खोज, प्रमाणीकरण और पूर्वावलोकन चलाना)। संक्षेप में, माइक्रोसर्विसेज एक एप्लिकेशन आर्किटेक्चर पैटर्न को संदर्भित करता है जो आमतौर पर  मोनोलिथिक एप्लिकेशन के विपरीत होता है।
+
+```
+
+#समस्या यह संबोधित करता है
+
+[Micro Services](https://github.com/cncf/glossary/blob/main/content/en/microservices.md) अखंड अनुप्रयोगों द्वारा उत्पन्न चुनौतियों की प्रतिक्रिया है। आम तौर पर, किसी एप्लिकेशन के विभिन्न हिस्सों को अलग-अलग स्केल करने की आवश्यकता होगी। एक ऑनलाइन स्टोर, उदाहरण के लिए, चेकआउट की तुलना में अधिक उत्पाद दृश्य होने वाला है। इसका मतलब है कि आपको चेकआउट की तुलना में चल रहे उत्पाद दृश्य कार्यक्षमता की अधिक प्रतियों की आवश्यकता होगी। एक अखंड अनुप्रयोग में, तर्क के उन बिट्स को अलग से तैनात नहीं किया जा सकता है। यदि आप व्यक्तिगत रूप से उत्पाद की कार्यक्षमता को माप नहीं सकते हैं, तो आपको पूरे ऐप को उन सभी घटकों के साथ डुप्लिकेट करना होगा जिनकी आपको आवश्यकता नहीं है - संसाधनों का एक अक्षम उपयोग। मोनोलिथिक एप्लिकेशन भी डेवलपर्स के लिए डिजाइन के नुकसान के आगे झुकना आसान बनाते हैं। चूंकि सभी कोड एक ही स्थान पर हैं, इसलिए उस कोड को कसकर युग्मित करना आसान है और चिंताओं को अलग करने के सिद्धांत को लागू करना कठिन है। मोनोलिथ को अक्सर डेवलपर्स को उत्पादक होने से पहले पूरे कोडबेस को समझने की आवश्यकता होती है।
```

Comment:
- "Microservices" is a single-word term.
- It seems that the link to the term itself is not necessary.

```suggestion
Microservices अखंड अनुप्रयोगों द्वारा उत्पन्न चुनौतियों की प्रतिक्रिया है। आम तौर पर, किसी एप्लिकेशन के विभिन्न हिस्सों को अलग-अलग स्केल करने की आवश्यकता होगी। एक ऑनलाइन स्टोर, उदाहरण के लिए, चेकआउट की तुलना में अधिक उत्पाद दृश्य होने वाला है। इसका मतलब है कि आपको चेकआउट की तुलना में चल रहे उत्पाद दृश्य कार्यक्षमता की अधिक प्रतियों की आवश्यकता होगी। एक अखंड अनुप्रयोग में, तर्क के उन बिट्स को अलग से तैनात नहीं किया जा सकता है। यदि आप व्यक्तिगत रूप से उत्पाद की कार्यक्षमता को माप नहीं सकते हैं, तो आपको पूरे ऐप को उन सभी घटकों के साथ डुप्लिकेट करना होगा जिनकी आपको आवश्यकता नहीं है - संसाधनों का एक अक्षम उपयोग। मोनोलिथिक एप्लिकेशन भी डेवलपर्स के लिए डिजाइन के नुकसान के आगे झुकना आसान बनाते हैं। चूंकि सभी कोड एक ही स्थान पर हैं, इसलिए उस कोड को कसकर युग्मित करना आसान है और चिंताओं को अलग करने के सिद्धांत को लागू करना कठिन है। मोनोलिथ को अक्सर डेवलपर्स को उत्पादक होने से पहले पूरे कोडबेस को समझने की आवश्यकता होती है।
```